app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    mv, file_name_1, file_name_2 = command.split()
+    if mv == "mv" and file_name_1 != file_name_2:
+        if '/' in file_name_2:
+            path = file_name_2[:file_name_2.rfind('/')]
+            os.makedirs(path)
+        with open(file_name_1, "r") as file_out, \
```

Comment:
Avoid using `\`. Use `( )`

Reply:
Hi. Could you please explain how to write `with (...)`. Because when I try, I get a syntaxerror - https://imgur.com/a/dVrGY4K.